sdk/cosmosdb/Azure.ResourceManager.CosmosDB/CHANGELOG.md

```diff
@@ -1,14 +1,10 @@
 # Release History
 
-## 1.4.0-beta.11 (Unreleased)
+## 1.4.0-beta.11 (2024-09-01)
```

Comment:
Content inside parathesis is a release date, not api version. You need to specify a future release date here.